e2e/nodejs_host/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {},
   "scripts": {
-    "postinstall": "node -e \"require('assert').equal(process.version, 'v12.12.0')\"",
+    "postinstall": "...",
```

Comment:
nit: I'd use this placeholder value more explicitly, it's not clear that `...` gets replaced. Maybe "placeholder" or "replaced during yarn_install"

Reply:
Yeah you're right, that's much more clear. I'll change and re-push.